Sources/SafeDIMacros/Macros/InjectableMacro.swift

```diff
@@ -141,7 +141,7 @@ public struct InjectableMacro: PeerMacro {
             case .fulfilledByTypeArgumentInvalidType:
                 "The argument `fulfilledByType` must be a string literal"
             case .fulfilledByTypeArgumentInvalidTypeDescription:
-                "The argument `fulfilledByType` must refer to a simple, unnested type"
+                "The argument `fulfilledByType` must refer to a simple type"
```

Comment:
I don't love the term "simple" here but I'm struggling to come up with better terminology.